EpiAware/src/EpiObsModels/ascertainment/helpers.jl

```diff
@@ -23,6 +23,6 @@ rand(gen_obs)
 "
 function ascertainment_dayofweek(model::AbstractTuringObservationModel;
         latent_model::AbstractTuringLatentModel = HierarchicalNormal(),
-        link = x -> exp.(x))
+        link = x -> x)
```

Comment:
Need to update the docs here to reflect changes to broadcast and the change in default.